src/main/java/com/dungeoncrawler/Javiarenka/shop/Armor_WIZARD.java

```diff
@@ -0,0 +1,10 @@
+package com.dungeoncrawler.Javiarenka.shop;
+
+import com.dungeoncrawler.Javiarenka.equipment.Armor;
+
+public class Armor_WIZARD implements BuyingArmorInterface {
+    @Override
+    public Armor getNewArmor() {
+        return null;
+    }
+}
```

Comment:
Ja bym chyba nie tworzył osobnych klas dla każdego HeroClassa, tylko jedną klasę do kupowania ekwipunku, który pobierałby listę wszystkich rzeczy, które można kupić i filtrował z nich te, które można kupić daną klasą. Teraz kod jest średnio przystosowany do rozszerzenia go (a to trochę niezgodne z zasadami programowania SOLID) - jak stwierdzimy kiedyś, ,że bedziemy chcieli zrobić jeszcze jedną, nową klasę herosa, to będziemy musieli napisać dużo nowego kodu.

Reply:
Zgadzam się z przedmówcą. Choć na początek pomysł niezły i działa :)